lib/internal/http2.js

```diff
@@ -1102,7 +1093,7 @@ function sessionOnStreamClose(stream, code) {
   setImmediate(maybeDestroyStream, stream);
 }
 
-function sessionOnError() {
+function sessionOnError(error) {
```

Comment:
It's used... see https://github.com/jasnell/http2-1/blob/7bb2fd9e88d103d999b6313f07b67c977eb2c890/lib/internal/http2.js#L1096-L1106

I added this because the code errors without it.